docs/contributing/code-reviewing.md

```diff
@@ -11,7 +11,7 @@ outside of the handful of people who do a ton of reviews -- and
 `@`-mention them with something like "`@person`, would you review
 this?". Good choices include
 
-- someone based in your timezone or a nearby timezone
+- someone based in your time zone or a nearby time zone
```

Comment:
this was correct, `timezone` is written together

Reply:
I believe "timezone" is not a word, though it's a reasonable variable name.